backend/src/main/java/de/neuefische/backend/controller/UserController.java

```diff
@@ -0,0 +1,19 @@
+package de.neuefische.backend.controller;
+
+import de.neuefische.backend.dto.UserResponseDto;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.security.Principal;
+
+@RestController
+@RequestMapping("api/user")
+public class UserController {
```

Comment:
Do you really need this controller?

Reply:
I wasn't sure and only added it at the end - but isn't it good to have so that I can test if login with valid token, expired token etc. works properly?

Reply:
You will be able to test that without this controller :)